dc-model-jackson/src/test/resources/serializedTestObjects/identifiable/IdentifierType.json

```diff
@@ -1,4 +1,5 @@
 {
+  "objectType" : "identifierType",
```

Comment:
Vielleicht auch besser in Großbuchstaben?

Reply:
done